src/components/Resources/submitResource.js

```diff
@@ -0,0 +1,261 @@
+        </Grid>
+
+        <Grid item lg={9}>
+          <div className={clsx(classes.root, classes.breadcrumb)}>
+            <Paper elevation={0} className={classes.paper}>
+              <Breadcrumbs separator={<NavigateNextIcon fontSize="small" />} aria-label="breadcrumb">
+                <Link color="inherit" href="#">
+                  Resources
+                </Link>
+                <Typography color="textPrimary">Submit</Typography>
+              </Breadcrumbs>
+            </Paper>
+          </div>
+          
+          <TextField
+            required
+            id="outlined-dense"
+            label="URL"
+            className={clsx(classes.textField, classes.dense)}
+            margin="dense"
+            variant="outlined"
+            fullWidth
+            value={values.url}
+            onChange={handleChange('url')}
+          />
+
+          <TextField
+            id="outlined-dense"
+            label="Title"
+            className={clsx(classes.textField, classes.dense)}
+            margin="dense"
+            variant="outlined"
+            fullWidth
+            value={values.title}
+            onChange={handleChange('title')}
+          />
+
+          <TextField
+            id="outlined-dense"
+            label="Description"
+            className={clsx(classes.textField, classes.dense)}
+            margin="dense"
+            variant="outlined"
+            fullWidth
+            value={values.description}
+            onChange={handleChange('description')}
+          />
+
+          <TextField
+            id="outlined-dense"
+            label="Year Updated"
+            className={clsx(classes.textField, classes.dense)}
+            margin="dense"
+            variant="outlined"
+            value={values.year}
+            onChange={handleChange('year')}
+          />
+          <FormGroup aria-label="position" row className={classes.freeSwitch}>
+            <FormControlLabel
+              checked={values.freeResource}
+              value="start"
+              control={<Switch color="primary" />}
+              label="Free"
+              labelPlacement="start"
+              onChange={handleSwitchChange('freeResource')}
+            />
+          </FormGroup>
+
+          <FormControl variant="outlined" className={classes.formControl}>
+            <InputLabel ref={inputLabel} htmlFor="outlined-media-type-native-simple" className={classes.label}>
+              Media Type
+            </InputLabel>
+            <Select
+              native
+              value={values.mediaType}
+              inputProps={{
+                name: 'media-type',
+                id: 'outlined-media-type-native-simple',
+              }}
+              className={classes.dense}
+              margin="dense"
+              labelWidth={labelWidth}
+              onChange={handleChange('mediaType')}
+            >
+              <option value="" />
+              <option value='pdf'>PDF</option>
```

Comment:
Could we add the following here too?

<option value='podcast'>Podcast</option>
<option value='tutorial'>Tutorial</option>
<option value='course'>Course</option>
<option value='book'>Book</option>
<option value='game'>Game</option>
<option value='blog'>Blog post</option>
<option value='event'>Event</option>
<option value='website'>Website</option>